src/v2/client/formatError.ts

```diff
@@ -55,6 +60,28 @@ export function formatInvalidRecoveryTokenError(error: StandardApiError) {
   return idxError;
 }
 
+export function formatInvalidActivationTokenError(error: StandardApiError) {
+  // This error comes from `oauth2/interact` so is not an IDX error.
+  // simulate an IDX-JS error response
+  const idxError = formatIDXError(error);
+  const { details } = idxError;
+  const messages: IdxMessages = {
+    type: 'array',
+    value: [
+      {
+        message: error.error_description,
+        i18n: {
+          key: 'idx.missing.activation.token'
+        },
+        class: 'ERROR'
+      }
```

Comment:
So I assume gen2 adds the "Activation link no longer valid" title elsewhere?

Reply:
Yes, in `TerminalView`
https://github.com/okta/okta-signin-widget/blob/b4090328bc1815cf6cff9d7caae1dc177151b6ae/src/v2/view-builder/views/TerminalView.js#L79
https://github.com/okta/okta-signin-widget/blob/b4090328bc1815cf6cff9d7caae1dc177151b6ae/src/v2/view-builder/views/TerminalView.js#L110-L118